### PR TITLE
[V1] Wallet Connect with Layout

### PR DESCRIPTION
**Pull request by mattgle**
## Summary

Feature branch for Wallet Connect with Layout.

Includes the followings PRs:

- #392 
- #395
- #396 
- #406
- #446
- #449 
- #450 
- #462
- #463
- #473
- #478

### CODE REVIEW

**Review by aleacevedo — APPROVED** [review]

File: src/screens/flows/WalletConnect/screens/Flows/components/RequestCall/index.tsx [hunk 1]
```diff
@@ -1,20 +1,130 @@
...
+  WCWhiteListItem,
+} from '@/interfaces/walletConnect';
+import KeyRing from '@/modules/keyring';
+import { getDabNfts, getDabToken } from '@/services/DAB';
+import { formatAssetBySymbol } from '@/utils/currencies';
+import { addSpacesAndCapitalize } from '@/utils/strings';
 
-import styles from '../../../InitialConnection/styles';
+// TODO: Pass this .png to .svg after 0.2.0 merge
+import warningIcon from '../../assets/warningIcon.png';
```
Comment (mattgle) [inline src/screens/flows/WalletConnect/screens/Flows/components/RequestCall/index.tsx:19]: We should do this

File: src/screens/flows/WalletConnect/screens/Flows/components/TransferItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { Image as FastImage } from '@/components/common';
+import Text from '@/components/common/Text';
+import TokenFormat from '@/components/formatters/TokenFormat';
+import UsdFormat from '@/components/formatters/UsdFormat';
+import Icon from '@/components/icons';
+import { FontStyles } from '@/constants/theme';
+
+// TODO: Pass .png to .svg after 0.2.0 merge
+import questionMark from '../../assets/questionMark.png';
+import transferRequest from '../../assets/transferRequest.png';
```
Comment (mattgle) [inline src/screens/flows/WalletConnect/screens/Flows/components/TransferItem/index.tsx:14]: same

File: src/screens/flows/WalletConnect/screens/Flows/components/RequestTransferHeader/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import { FontStyles } from '@/constants/theme';
+
+import UserShowcase from '../UserShowcase';
+import styles from './styles';
+
+interface Props {
+  canisterId: string;
+}
+
+function RequestTransferHeader(props: Props) {
```
Comment (mattgle) [inline src/screens/flows/WalletConnect/screens/Flows/components/RequestTransferHeader/index.tsx:16]:
```suggestion
function RequestTransferHeader({ canisterId }: Props) {
```

File: src/screens/flows/WalletConnect/screens/TimeoutError/index.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+interface Props {
+  route: {
+    params: {
+      dappUrl: string;
+      dappName: string;
+    };
+  };
+}
+
+function WCTimeoutError({ route }: Props) {
```
Comment (mattgle) [inline src/screens/flows/WalletConnect/screens/TimeoutError/index.tsx:25]: import navigation from props

**Pull request merged**